protos/fpc/attestation.proto

```diff
@@ -0,0 +1,21 @@
+// Copyright IBM Corp. All Rights Reserved.
+// Copyright 2020 Intel Corporation
+//
+// SPDX-License-Identifier: Apache-2.0
+
+syntax = "proto3";
+
+package attestation;
+option go_package = "github.com/hyperledger-labs/fabric-private-chaincode/internal/protos";
```

Comment:
given that we modularlize attestation (and tl_session), i wonder whether we might not put the protobufs into module-specific packages?
```suggestion
option go_package = "github.com/hyperledger-labs/fabric-private-chaincode/internal/protos/attestation";
```

Reply:
oh if we do that we need to move the generated protos also to corresponding subfolders such as `internal/protos/attestation` etc ... otherwise we have multiple go packages in a single folder :/

Reply:
> oh if we do that we need to move the generated protos also to corresponding subfolders such as `internal/protos/attestation` etc ... otherwise we have multiple go packages in a single folder :/
> 
> Edit: just tested, protoc will not create these folders for us.

I would have hoped that protoc/nanopb would do create the folders.  As not, let's leave it as is ...
(Judging from https://github.com/golang/protobuf/issues/515 it seems though as iff at least for go the default is/was to generate it in the proper directories?)

Reply:
cool this works! updated